packages/user-group-service/src/groups/typedef.graphql

```diff
@@ -30,13 +30,34 @@ type Mutation {
 
 scalar DateTime
 
+type UserType {
```

Comment:
Isn't this type already defined in the `users/typedef.graphql`?

Why does it need to be defined again? GraphQL stitches all the typedefs together and replaces any duplicate types, so I would not recommend redefining it again.

Reply:
Removed